doc/full-reindex-process.md

```diff
@@ -0,0 +1,22 @@
+
+# Full Reindex Process Steps
+
+This document outlines the expected steps needed to complete the Full Reindex Process to go from exporting data from Alma to having it live in the Production Library Search. This is intended as a checklist for the person responsible for shepherding theprocess to ensure that the steps that the required manual steps are followed.
+
+1. Initiate Alma FTP Export and wait for completion.
+1. Trigger the `catalog_move_alma_sftp_to_s3` Dag and wait for completion.
+1. Set variables required for the full reindex:
+   1. Set the variable `ALMASFTP_S3_ORIGINAL_DATA_NAMESPACE` to the folder name the dag used as a namespace, which will appear in the dags Success slack post. 
```

Comment:
This looks good, but just wondering if there's an alternate way of getting this variable as in cases where we want to reuse the SFTP dump from a previous full reindex (like we're about to do)?